docker/fullstack/entry.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
 
+export FRONTEND_URL="${FRONTEND_URL:-${HYPERDX_APP_URL:-http://localhost}:${HYPERDX_APP_PORT:-8080}}"
+
+echo "Visit the HyperDX UI at $FRONTEND_URL"
+echo ""
+
 # Use concurrently to run both the API and App servers
 npx concurrently \
   "--kill-others" \
   "--names=API,APP" \
-  "APP_TYPE=api PORT=${HYPERDX_API_PORT:-8000} node -r /app/api/node_modules/@hyperdx/node-opentelemetry/build/src/tracing /app/api/build/index.js" \
-  "/app/app/node_modules/.bin/next start -p ${HYPERDX_APP_PORT:-8080}"
+  "PORT=${HYPERDX_API_PORT:-8000} HYPERDX_APP_PORT=${HYPERDX_APP_PORT:-8080} node -r /app/api/node_modules/@hyperdx/node-opentelemetry/build/src/tracing /app/api/build/index.js" \
```

Comment:
can follow up in another pr, but can we add concurrently to dependencies so we don't get `npm warn exec The following package was not found and will be installed: concurrently@9.1.0` on boot? it also makes new boots really slow

Reply:
Good point. Let me address it in this PR